src/stmpe1600_class.h

```diff
@@ -0,0 +1,178 @@
+/**

```

Comment:
Why a gpio expander class is in this lib?

Reply:
We kept the sources as was. Maybe it is an old compatibility for ST Nucleo board (not the stm32duino project). It seems we can delete it but I prefer to ask to @cparata (AST team).

Reply:
Original code was for a dedicated expansion board. So, this should be removed as the Arduino lib only manage one component. Dedicated library could be created later to support the gpio expander (pe1600) which is used by several expansion board.
@cparata do no hesitate to comment.

Reply:
Thanks you @cparata for your fix (in backstage ^^). The pe1600 dependencies has been removed from this library.